dask_sphinx_theme/static/css/style.css

```diff
@@ -288,12 +294,9 @@ ul.current.nav.bd-sidenav
 }
 
 footer {
-  bottom: 0;
-  left: 0;
-  width: 76%;
-  height: 50px;
   background-color: var(--grey);
```

Comment:
Thanks @scharlottej13! 

To me having a dark footer background seems a bit off. I'm curious if this was intentional or if we should switch to something lighter?

Reply:
Funny you should ask.... I first opened this PR removing the grey background entirely because I thought it looked better, but the design docs suggested a darker footer. Here's w/ the white background:


![Screen Shot 2022-06-09 at 1 27 43 PM](https://user-images.githubusercontent.com/8620816/172941971-13292d52-760e-41f3-81a4-4aaa8dd14ca5.png)

Reply:
Just to chime in. I think the dark footer looks wierd because it doesn't span the whole width of the page. I think an exception should be made for this case and we should do the white footer.

Reply:
That sounds great to me too. I'll update the PR.